src/plasoscaffolder/bll/templates/formatter_template.jinja2

```diff
@@ -1,27 +1,29 @@
 # -*- coding: utf-8 -*-
-"""TODO"""
+"""{{ plugin_name|replace('_',' ')}} database formatter."""
 
 from plaso.formatters import interface
 from plaso.formatters import manager
-from plaso.lib import errors
+from plaso.lib import errors{% for query in queries %}
```

Comment:
actually not.
This is because i do not want a newline there.
If i put {% for.... on the next line i get a newline infront of the class, but only the first class for the queries. i want the same amount of newline infront of each class.
But yes it does look a bit silly in the template.